src/services/authorization.ts

```diff
@@ -190,9 +190,14 @@ export const validatePermission = async (
 /**
  * Internal filtering function that takes out limited items (eg. hidden children)
  *  */
-const _filterOutItems = async (actor: Actor, repositories: Repositories, items: Item[]) => {
+const _filterOutItems = async (
```

Comment:
If this function should be private, we should enforce it.

```suggestion
private const filterOutItems = async (
```

Reply:
Because of this function is not part of a class, I won't be able to add the private clause. We can encapsulate the whole file into a static class, but it would need a lot of changes.

Reply:
Ah I didn't saw correctly, my bad ^^. But maybe we don't need the underscore because the function is not exported ? What do you think ?

Reply:
It looks like there's already a `filterOutItems` function  in the same file which basically call the private function. So we would have to rename something. I agree that the `_` could not be necessary. Any suggestions ?

What about `filterOutItemsByMembershipAndTags` ? It seems too verbose to me. Maybe the function is doing too much things ? What do you think @spaenleh & @pyphilia ?

Reply:
As we discussed with Thibault this morning. Since we don't export the function we won't be able to use it outside the file. But I understand the use of `_` here, because even if the function is not exported in its declaration, it still can be exported at the end of the file.

If there's no better solution, I won't modify the name since it doesn't concern this PR. If needed we can open another PR to refactor the `authorization.ts` structure ?